From 7437108a3c522710ed2da3c018d938878e51084a Mon Sep 17 00:00:00 2001
From: Dylan Van Assche <me@dylanvanassche.be>
Date: Thu, 3 Dec 2020 07:52:00 +0100
Subject: [PATCH] panels/info-overview: Use GNOME logo instead of Purism

pmOS should not ship this with a Purism logo since pmOS uses upstream as much as possible.
---
 panels/info-overview/cc-info-overview-panel.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panels/info-overview/cc-info-overview-panel.ui b/panels/info-overview/cc-info-overview-panel.ui
index 4af20ccba..bd43be8c9 100644
--- a/panels/info-overview/cc-info-overview-panel.ui
+++ b/panels/info-overview/cc-info-overview-panel.ui
@@ -30,7 +30,7 @@
                   <object class="GtkImage">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
-                    <property name="resource">/org/gnome/control-center/info-overview/purism-logo.svg</property>
+                    <property name="resource">/org/gnome/control-center/info-overview/GnomeLogoVerticalMedium.svg</property>
                   </object>
                   <packing>
                     <property name="expand">False</property>
-- 
2.25.1

